src/applications/personalization/preferences/tests/helpers.unit.spec.jsx

```diff
@@ -101,4 +105,30 @@ describe('transformPreferencesForSaving', () => {
       expect(filteredList.length).to.equal(1);
     });
   });
+  describe('didPreferencesChange', () => {
+    it('should return `true` if the `preferences` do not match', () => {
+      const prevProps = { preferences: { key: { key: true } } };
+      const nextProps = { preferences: { key: { key: false } } };
+      expect(didPreferencesChange(prevProps, nextProps)).to.equal(true);
+    });
+    it('should return `false` if the `preferences` match', () => {
+      const prevProps = { preferences: { key: { key: true } } };
+      const nextProps = { preferences: { key: { key: true } } };
+      expect(didPreferencesChange(prevProps, nextProps)).to.equal(false);
+    });
+  });
+  describe('didJustSave', () => {
```

Comment:
I couldn't find the implementation for `didJustSave()`

Reply:
It's there https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/personalization/preferences/helpers.jsx#L273

Reply:
Thanks I'm just blind. Looks good.